src/main/java/swinglib/JTextFieldBuilder.java

```diff
@@ -126,8 +126,7 @@ public static void keyTypedListener(final Consumer<KeyEvent> listener, final JTe
     Preconditions.checkArgument(fields.length > 0);
     Preconditions.checkNotNull(fields[0]);
 
-    Arrays.asList(fields).forEach(field ->
-        field.addKeyListener(SwingAction.keyReleaseListener(listener)));
+    Arrays.asList(fields).forEach(field -> field.addKeyListener(SwingAction.keyReleaseListener(listener)));
```

Comment:
Nothing that it's probably faster to use Arrays.stream instead of creating a list